.github/workflows/dh-backend-ci.yml

```diff
@@ -11,6 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Continuous Integration of domain except for own NuGet packages
```

Comment:
nit: We usually only have this comment in repositories that has a NuGet package, which I don't believe this one has